python/ray/_raylet.pyx

```diff
@@ -207,6 +199,21 @@ def compute_task_id(ObjectID object_id):
     return TaskID(object_id.native().TaskId().Binary())
 
 
+cdef increase_recursion_limit():
```

Comment:
Maybe we can just initially set the recursion limit to `20 * max_concurrency`? Might be a bit simpler

Reply:
That won't quite work because we don't know how many stack is there per actor call. The doubling approach is less brittle